purchase_order_type/__openerp__.py

```diff
@@ -1,46 +0,0 @@
-# coding: utf-8
-###############################################################################
-#    Module Writen to OpenERP, Open Source Management Solution
-#    Copyright (C) OpenERP Venezuela (<http://www.vauxoo.com>).
-#    All Rights Reserved
-############# Credits #########################################################
-#    Coded by: Yanina Aular <yanina.aular@vauxoo.com>
-#    Planified by: Humberto Arocha <hbto@vauxoo.com>
-#    Audited by: Humberto Arocha <hbto@vauxoo.com>
-###############################################################################
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-###############################################################################
-{
-    "name": "Purchase Order Type",
-    "version": "8.0.0.1.6",
-    "author": "Vauxoo",
-    "category": "purchase",
-    "website": "http://www.vauxoo.com/",
-    "license": "",
-    "depends": [
-        "purchase",
-        "purchase_requisition_type",
-        "pr_line_related_po_line"
-    ],
-    "demo": [],
-    "data": [
-        "view/purchase_order_view.xml"
-    ],
-    "test": [],
-    "js": [],
-    "css": [],
-    "qweb": [],
-    "installable": False,
```

Comment:
This module is not installable. Did it cause conflict anyway?

Reply:
yes, because the order in addons paths

>>en instancias de deploy, este modulo tomaba prioridad encima del de purchase-workflow por el orden en los addons-path

Esto causaba el conflicto